arwenmandos/elrondProtectedStorage.go

```diff
@@ -1,7 +1,7 @@
 package arwenmandos
 
-// ElrondProtectedKeyPrefix prefixes all Elrond reserved storage. Only the protocol can write to keys starting with this.
-const ElrondProtectedKeyPrefix = "ELROND"
+// ProtectedKeyPrefix prefixes all Elrond reserved storage. Only the protocol can write to keys starting with this.
+const ProtectedKeyPrefix = "ELROND"
```

Comment:
please remove this and use instead mx-chain-core-go/core/constants.go `ProtectedKeyPrefix`